presto-parser/src/main/java/com/facebook/presto/sql/SqlFormatter.java

```diff
@@ -912,6 +919,16 @@ private String formatSqlParameterDeclarations(List<SqlParameterDeclaration> para
                     .collect(joining(",\n", "(\n", "\n)"));
         }
 
+        private static String formatTypeList(List<String> types, Integer indent)
+        {
+            String elementIndent = indentString(indent + 1);
+            return format(
+                    "(\n%s)\n",
```

Comment:
I prefer to show this all in one line, including the function name. That feels more natural to me for SQL functions. Don't have super strong opinions though. But I think most of function input would fit into one line fine.